packages/components/src/modal/index.tsx

```diff
@@ -283,7 +289,17 @@ function UnforwardedModal(
 								) }
 							</div>
 						) }
-						<div ref={ childrenContainerRef }>{ children }</div>
+						<div ref={ childrenContainerRef }>
+							<div
+								ref={
+									focusOnMount === 'firstContentElement'
+										? focusOnMountRef
+										: noopRef
+								}
```

Comment:
I guess we could just not add the ref at all. Does passing `undefined` work with refs?

Reply:
It does. Updated.